pkg/jira/client_test.go

```diff
@@ -0,0 +1,41 @@
+package jira
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getIssue(t *testing.T) {
```

Comment:
This test for `GetAllIssues` method right?

Reply:
yes, I forgot to rename it, thanks!